haskell/src/Monocle/Backend/Index.hs

```diff
@@ -206,6 +220,9 @@ createIndex indexName mapping = do
   where
     indexSettings = BH.IndexSettings (BH.ShardCount 1) (BH.ReplicaCount 0)
 
+schemaVersion :: ConfigVersion
```

Comment:
nit: perhaps this should be named `configVersion`, or `latestConfigVersion` ?

Reply:
Thanks see next commit.